cmd/osv-scanner/main.go

```diff
@@ -89,6 +89,11 @@ func run(args []string, stdout, stderr io.Writer) int {
 				Usage:   "check subdirectories",
 				Value:   false,
 			},
+			&cli.BoolFlag{
+				Name:  "call-analysis",
```

Comment:
Can we include `--experimental` in the flag name itself to make its experimental state more explicit?